### PR TITLE
add apache license

### PR DESCRIPTION
**Pull request by tennix**

### CODE REVIEW

File: docker/dashboard_installer/build.sh [hunk 1]
```diff
@@ -14,4 +14,5 @@ do
     destFile=`echo ${mtpl}|awk -F"[-.]" '{print $1"."$3}'`
     cp config/${mtpl} docker/dashboard_installer/dashboard/${destFile}
 done
+cp LICENSE docker/dashboard_installer/dashboard/LICENSE
```
Comment (gregwebs) [inline docker/dashboard_installer/build.sh]: you could `rm` this file at the end of the build.

File: docker/dashboard_installer/dashboard/LICENSE [hunk 2]
```diff
@@ -0,0 +1,201 @@
+Apache License
```
Comment (onlymellb) [inline docker/dashboard_installer/dashboard/LICENSE]: This file should be written to `.gitignore` file to ignore

**Comment by onlymellb** [comment]
Rest LGTM

**Comment by tennix** [comment]
I think we should re-organize dashboard-installer to avoid copying json files from config directory. I'll send another PR for this.

**Pull request merged**